hkmc2/shared/src/main/scala/hkmc2/utils/utils.scala

```diff
@@ -36,11 +37,16 @@ extension (t: Product)
       case Nil => "Nil"
       case xs: List[_] => "Ls of \n" + xs.iterator.map(aux(_)).mkString("\n").indent("  ")
       case s: String => s.escaped
-      case FldFlags(mut, spec, genGetter) =>
+      case TermDefFlags(mod) =>
+        val flags = Buffer.empty[String]
+        if mod then flags += "module"
+        if flags.isEmpty then "()" else flags.mkString("(", ", ", ")")
```

Comment:
`if` is unnecessary.

Reply:
I was thinking that it can be easily extended for future, so I just copied those from below. You don't like it? 🫡

Reply:
I mean the `if` is completely redundant; you can get the same result by:
```suggestion
        flags.mkString("(", ", ", ")")
```

Reply:
I just copied it from somewhere below... 🥲 Anyway will fix it

Reply:
Ah yes, this appears to be @chengluyu's [beautiful work](https://github.com/hkust-taco/mlscript/commit/b9190b3630c1979a23953d575fcd78bbeb312dc1#diff-e59b35cf20bc0a11500f04c2828d5134dbee399ce45f1e42c1255261722ab67cR36). Please also fix the original. The idea is that one should strive to leave the code in a better place than they found it, if possible.